data/src/main/java/com/microsoft/azure/kusto/data/AadAuthenticationHelper.java

```diff
@@ -8,18 +8,21 @@
 
 import javax.naming.ServiceUnavailableException;
 import java.awt.*;
-import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.security.PrivateKey;
 import java.security.cert.X509Certificate;
+import java.util.Date;
 import java.util.concurrent.*;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
 
 public class AadAuthenticationHelper {
 
     private final static String DEFAULT_AAD_TENANT = "common";
     private final static String CLIENT_ID = "db662dc1-0cfe-4e1c-a843-19a68e65be58";
+    public final static long ONE_MINUTE_IN_MILLIS = 60000;
```

Comment:
```suggestion
    final static long ONE_MINUTE_IN_MILLIS = 60000;
```

Reply:
Well, thinking about this again, I think you should rename this according to the usage, something like:
MIN_ACCESS_TOKEN_VALIDITY_IN_MILLISECS